.travis.yml

```diff
@@ -538,8 +537,8 @@ before_script:
   - tar xf cmocka-1.1.0.tar.xz
   - cd cmocka-1.1.0
   - mkdir build
-  - cd build
```

Comment:
You removed `cd build`, without it `-DWITH_STATIC_LIB=true` cannot wotk

Reply:
Yeah, I noticed that too... I have no idea why (and even when) I did it...